### PR TITLE
Exercise 1678 and 1720

### PR DESCRIPTION
**Pull request by JAIRMG**
Exercise 1720 is pretty slow :(

### CODE REVIEW

File: Easy/1720 Decode XORed Array.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ Output: [1,0,2,1]
+ Explanation: If arr = [1,0,2,1], then first = 1 and encoded = [1 XOR 0, 0 XOR 2, 2 XOR 1] = [1,2,3]
+ 
+ */
+
+//Runtime: 320 ms, faster than 17.05% of Swift online submissions for Decode XORed Array.
+//Memory Usage: 15.5 MB, less than 20.45% of Swift online submissions for Decode XORed Array.
+
+
+func decode(_ encoded: [Int], _ first: Int) -> [Int] {
```
Comment (LeonardoDzo) [inline Easy/1720 Decode XORed Array.playground/Contents.swift:37]: My solution was a little faster than yours but I think it still slow yet.
_Runtime: 276 ms, faster than 42.05% of Swift online submissions for Decode XORed Array.
Memory Usage: 15.2 MB, less than 87.50% of Swift online submissions for Decode XORed Array._

```suggestion
 func decode(_ encoded: [Int], _ first: Int) -> [Int] {
        var lastValue = first
        var result = encoded.map { e -> Int in 
            let value = e ^ lastValue
            lastValue = value
            return value
        }
        result.insert(first, at: 0)
        return result
    }
```

**Review by LeonardoDzo — APPROVED** [review]

File: Easy/1678 Goal Parser Interpretation.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+//Runtime: 0 ms, faster than 100.00% of Swift online submissions for Goal Parser Interpretation.
+//Memory Usage: 15.9 MB, less than 12.31% of Swift online submissions for Goal Parser Interpretation.
+func interpret(_ command: String) -> String {
+
+    var result = ""
+
+    result += command.replacingOccurrences(of: "()", with: "o")
```
Comment (LeonardoDzo) [inline Easy/1678 Goal Parser Interpretation.playground/Contents.swift:30]: Nice

**Review by LeonardoDzo — APPROVED** [review]

File: Easy/1720 Decode XORed Array.playground/Contents.swift (hunk 1 above)
Comment (JAIRMG) [reply Easy/1720 Decode XORed Array.playground/Contents.swift:37]: will commit your solution on this exercise 👍

Comment (JAIRMG) [reply Easy/1720 Decode XORed Array.playground/Contents.swift:37]: Wait, I just ran my solution and now is giving me 
```
Runtime: 264 ms, faster than 84.09% of Swift online submissions for Decode XORed Array.
Memory Usage: 15.3 MB, less than 47.73% of Swift online submissions for Decode XORed Array.
```

Yours is giving me 

```
Runtime: 268 ms, faster than 70.45% of Swift online submissions for Decode XORed Array.
Memory Usage: 15.5 MB, less than 32.95% of Swift online submissions for Decode XORed Array.
```
not much difference, both are similar solutions 👍

**Pull request merged**